projects/OG-Analytics/src/main/java/com/opengamma/analytics/financial/provider/description/interestrate/MulticurveProviderDiscount.java

```diff
@@ -57,15 +58,15 @@
   /**
    * Map of curve names to currencies.
    */
-  private Map<String, Currency> _namesToCurrency;
+  private Map<String, List<Currency>> _namesToCurrency;
```

Comment:
Those objects are used to loop on curves in different cases. The order is important is several cases.